src/core/action.ts

```diff
@@ -83,9 +83,14 @@ export abstract class SchedulableAction<T extends Present> implements Sched<T> {
                 if (intendedTag === undefined) {
                     throw new Error("FederatedPortAction must have an intended tag from RTI.");
                 }
-                if (intendedTag <= this.action.runtime.util.getCurrentTag()) {
+                if (!this.action.runtime.util.isLastTagProvisional() && intendedTag <= this.action.runtime.util.getCurrentTag()) {
```

Comment:
Should it also be `isLastTAGProvisional`?

Reply:
Done!